tests/features/ecas-login.feature

```diff
@@ -43,17 +44,13 @@ Feature: Login through OE Authentication
     And I should see the link "Log in"
 
   Scenario: A blocked user should not be able to log in
```

Comment:
The scenario name no longer reflects what we test. Now we test that by default, any new users are created as blocked.

Reply:
Updated to "A site that requires administration validation on users should block them by default", marking the conversation as resolved.